packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -1937,18 +1938,29 @@ class SelectableRegionState extends State<SelectableRegion>
     if (_webContextMenuEnabled) {
       result = PlatformSelectableRegionContextMenu(child: result);
     }
-    return CompositedTransformTarget(
-      link: _toolbarLayerLink,
-      child: RawGestureDetector(
-        gestures: _gestureRecognizers,
-        behavior: HitTestBehavior.translucent,
-        excludeFromSemantics: true,
-        child: Actions(
-          actions: _actions,
-          child: Focus.withExternalFocusNode(
-            includeSemantics: false,
-            focusNode: _focusNode,
-            child: result,
+    return TapRegion(
+      groupId: SelectableRegion,
+      onTapOutside: (PointerDownEvent event) {
+        // To match the native web behavior, this selectable region is
```

Comment:
Is this behavior the same on mobile and desktop?

Reply:
I checked WinUI 3 on Windows, it also does not dismiss the text selection if you click outside of it (unless your click moves the focus).

In my initial review, I also wondered if this was correct. It might be worth updating the comment to clarify that this is indeed intentional. Maybe something like:

```
Tapping outside of the text selection does not dismiss it on Android, macOS, and Windows.
```